### PR TITLE
Propose changes to the Certificate of Incorporation

### PR DESCRIPTION
**Pull request by arecvlohe**
## Background

I am proposing changes to the certificate of incorporation to keep things up to date such as my latest address and directors just to name a few.

### CODE REVIEW

**Comment by erinishimoticha** [comment]
Where does the Certificate of Incorporation canonically reside right now and why are we amending it by adding a new file to this repo?

**Comment by arecvlohe** [comment]
@erinishimoticha It was in google drive but I needed to make it searchable by everyone: https://drive.google.com/file/d/1yUpQ-uSFgbAHK7XcQo-d4o2UxTmnqK5c/view?usp=sharing. The State requires that we approve changes at a board meeting but I figured we should just follow our Git protocol for decisions like this.

```
At a meeting of the governing body of said corporation, a resolution was duly adopted setting forth the foregoing proposed amendment(s) to the certificate of incorporation of said corporation, declaring said amendment(s) to be advisable and calling a meeting of the members for consideration thereof.
Pursuant to such call and to due written notice given to each member, a meeting was held, at which meeting the necessary number of members as required by the certificate of incorporation of said corporation voted in favor of the amendment(s).
```

**Comment by csutherl** [comment]
I'm not crazy about sharing information about my physical location anywhere that's public and easily searchable. I think we should keep the nonprofit certificate private and omit those private details from any change logs that we're recording in regular meeting minutes. I'm not sure how to accomplish that (maybe having special meetings to conduct sensitive business?) and be super transparent at the same time, but some info has to remain private IMO.

**Comment by erinishimoticha** [comment]
I, too, would prefer that my address is not public here linked to my github and twitter and everything else. Can you rebase those specific details out of this PR?

File: Amed_Certificate_of_Incorporation-20200714.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Amend Certificate of Incorporation 20200714
```
Comment (erinishimoticha) [inline Amed_Certificate_of_Incorporation-20200714.md:1]: Rename the file so "Amend" is spelled correctly.

**Pull request closed without merging**